Wikipedia/Localizations/tr.lproj/Localizable.strings

```diff
@@ -736,19 +738,28 @@
 "table-of-contents-close-accessibility-label" = "İçindekileri kapat";
 "table-of-contents-heading" = "Konu başlıkları";
 "table-title-other" = "Daha fazla bilgi";
+"talk-page-add-discussion-accessibility-label" = "Tartışma ekle";
+"talk-page-discussion-accessibility-hint" = "Tartışma başlığını açmak için iki kez hafifçe vurun";
+"talk-page-discussion-read-accessibility-label" = "Oku";
+"talk-page-discussion-read-ipa-accessibility-attribute" = "rɛd";
```

Comment:
Note flagged this as potentially incorrect [here](https://translatewiki.net/wiki/Support#IPA_pronunciation_for_ios-talkpage-discussion_read.2Funread_strings_.28TR.29_56584).